src/commands/generate/__tests__/resources/test-cases/example-types/product/emptyEnum.ts

```diff
@@ -0,0 +1,3 @@
+export const EmptyEnum = {};
```

Comment:
does this still allow you to iterate the keys of the empty enum?

Reply:
`namespace EmptyEnum {}` does not, but `const EmptyEnum = {}` does (because it's just a normal object)